purefa/metadata.csv

```diff
@@ -0,0 +1,37 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name

```

Comment:
We recently added a new column called `curated_metrics`. Example: https://github.com/DataDog/integrations-extras/pull/1210. Can you add that to your metadata.csv?

Reply:
Ack